num_points: 289
total_length_m: 1890.3113222346783
loop_path: false
path:
- x: -17.101009368896484
  y: -6.0
  yaw_rad: 0.0036213111576023607
  yaw_deg: 0.20748584563424977
- x: -13.10626220703125
  y: -5.985533714294434
  yaw_rad: 2.3573490164076683
  yaw_deg: 135.06614947947526
- x: -17.101009368896484
  y: -2.0
  yaw_rad: -0.003614104390001304
  yaw_deg: -0.20707292826677762
- x: -13.106608390808105
  y: -2.0144362449645996
  yaw_rad: 0.14496385861712635
  yaw_deg: 8.30581728069251
- x: 4.3259596824646
  y: 0.5305080413818359
  yaw_rad: -1.2067083926596596
  yaw_deg: -69.13929800241382
- x: 6.2398271560668945
  y: -4.491748809814453
  yaw_rad: -2.095626784792866
  yaw_deg: -120.07057020320167
- x: 3.546757459640503
  y: -9.14304256439209
  yaw_rad: -2.9845355059931395
  yaw_deg: -171.0012883003485
- x: -1.761547327041626
  y: -9.983673095703125
  yaw_rad: 2.4097412677455887
  yaw_deg: 138.06800436032674
- x: -5.759976863861084
  y: -6.392055034637451
  yaw_rad: 1.520832532374295
  yaw_deg: 87.13728545124025
- x: -5.491560459136963
  y: -1.0243079662322998
  yaw_rad: 0.15706707026225694
  yaw_deg: 8.999280226512083
- x: 4.3259596824646
  y: 0.5305080413818359
  yaw_rad: -1.2067083926596596
  yaw_deg: -69.13929800241382
- x: 6.2398271560668945
  y: -4.491748809814453
  yaw_rad: -2.095626784792866
  yaw_deg: -120.07057020320167
- x: 3.546757459640503
  y: -9.14304256439209
  yaw_rad: -2.9845355059931395
  yaw_deg: -171.0012883003485
- x: -1.761547327041626
  y: -9.983673095703125
  yaw_rad: 2.518752512323417
  yaw_deg: 144.31388859410467
- x: -5.639927864074707
  y: -7.198199272155762
  yaw_rad: 2.71787532351296
  yaw_deg: 155.72278528004583
- x: -8.21687126159668
  y: -6.035898208618164
  yaw_rad: 2.001395606981347
  yaw_deg: 114.67152141605482
- x: -10.064788818359375
  y: -2.0129714012145996
  yaw_rad: 0.1520054348733567
  yaw_deg: 8.70926988129404
- x: -6.751442909240723
  y: -1.505409598350525
  yaw_rad: 0.6011457379248528
  yaw_deg: 34.44311365537154
- x: -4.200197696685791
  y: 0.2442857027053833
  yaw_rad: 0.3258680655029224
  yaw_deg: 18.67086483141011
- x: 1.2457516193389893
  y: 2.0845518112182617
  yaw_rad: -0.6196887005441944
  yaw_deg: -35.50554715312866
- x: 5.023921966552734
  y: -0.6109428405761719
  yaw_rad: -0.40606076598152363
  yaw_deg: -23.265568116590696
- x: 7.894852638244629
  y: -1.8453161716461182
  yaw_rad: -3.132257814686497
  yaw_deg: -179.4651531284066
- x: -10.064788818359375
  y: -2.0129714012145996
  yaw_rad: 0.3666393613363651
  yaw_deg: 21.006888007945694
- x: -7.2257866859436035
  y: -0.9227889776229858
  yaw_rad: 0.5239214567969471
  yaw_deg: 30.018488270810767
- x: -3.762331008911133
  y: 1.0783284902572632
  yaw_rad: 0.06983373644608508
  yaw_deg: 4.001178365989592
- x: 1.8205511569976807
  y: 1.468837022781372
  yaw_rad: -0.852601152492662
  yaw_deg: -48.85044764581944
- x: 5.2356719970703125
  y: -2.439161539077759
  yaw_rad: -1.8208746278491887
  yaw_deg: -104.32843119821295
- x: 3.728517532348633
  y: -8.339729309082031
  yaw_rad: -2.9038318836291164
  yaw_deg: -166.37731134747239
- x: -1.6163828372955322
  y: -9.635037422180176
  yaw_rad: 2.5802044468819294
  yaw_deg: 147.83482508722156
- x: -5.7497029304504395
  y: -7.035654067993164
  yaw_rad: 2.756210632900462
  yaw_deg: 157.91923671427796
- x: -8.152029037475586
  y: -6.061110019683838
  yaw_rad: -1.6062038402090308
  yaw_deg: -92.02870108168274
- x: -9.0
  y: -30.0
  yaw_rad: 0.02121625161703204
  yaw_deg: 1.215601674743544
- x: -4.700081825256348
  y: -29.90875816345215
  yaw_rad: 0.49131086274807495
  yaw_deg: 28.150038864395952
- x: -0.06710219383239746
  y: -27.42977523803711
  yaw_rad: 1.1696155202742857
  yaw_deg: 67.01403296471454
- x: 1.9848130941390991
  y: -22.592472076416016
  yaw_rad: 1.1172444123343785
  yaw_deg: 64.01338951133378
- x: 10.080679893493652
  y: -5.983635425567627
  yaw_rad: -3.0275235893953085
  yaw_deg: -173.4643240486492
- x: 6.614617824554443
  y: -6.380729675292969
  yaw_rad: -2.627248674227112
  yaw_deg: -150.53026076455447
- x: 3.8615453243255615
  y: -7.936424255371094
  yaw_rad: -2.8979782465391692
  yaw_deg: -166.04192264741715
- x: -1.6761057376861572
  y: -9.312812805175781
  yaw_rad: 2.2740537739633426
  yaw_deg: 130.29368363389642
- x: -5.391779899597168
  y: -4.9304609298706055
  yaw_rad: 1.242741093905184
  yaw_deg: 71.20381970823816
- x: -3.5663905143737793
  y: 0.4327571392059326
  yaw_rad: 0.37500940811071054
  yaw_deg: 21.486456362442777
- x: 1.8119536638259888
  y: 2.549874782562256
  yaw_rad: -0.6117553030273668
  yaw_deg: -35.050996958214874
- x: 5.4896745681762695
  y: -0.03017449378967285
  yaw_rad: -0.5346406323671735
  yaw_deg: -30.632651790844477
- x: 8.299739837646484
  y: -1.6942046880722046
  yaw_rad: -1.541993874467936
  yaw_deg: -88.3497410420384
- x: 9.0
  y: -26.0
  yaw_rad: -1.5707963267948966
  yaw_deg: -90.0
- x: 9.0
  y: -30.0
  yaw_rad: 0.7853981633974483
  yaw_deg: 45.0
- x: 13.0
  y: -26.0
  yaw_rad: -1.5707963267948966
  yaw_deg: -90.0
- x: 13.0
  y: -30.0
  yaw_rad: 0.7853981633974483
  yaw_deg: 45.0
- x: 17.0
  y: -26.0
  yaw_rad: -1.5707963267948966
  yaw_deg: -90.0
- x: 17.0
  y: -30.0
  yaw_rad: 0.7853981633974483
  yaw_deg: 45.0
- x: 21.0
  y: -26.0
  yaw_rad: -1.5707963267948966
  yaw_deg: -90.0
- x: 21.0
  y: -30.0
  yaw_rad: 0.6302322273176791
  yaw_deg: 36.109646738432524
- x: 25.725929260253906
  y: -26.552574157714844
  yaw_rad: -1.9707967341548929
  yaw_deg: -112.91833514524146
- x: 24.168254852294922
  y: -30.23681640625
  yaw_rad: -0.04229702929601475
  yaw_deg: -2.4234412646028445
- x: 29.524272918701172
  y: -30.4634952545166
  yaw_rad: -2.7707960744093607
  yaw_deg: -158.75492095507278
- x: 25.79611587524414
  y: -31.912927627563477
  yaw_rad: -0.6555961378451016
  yaw_deg: -37.56289176360127
- x: 30.0
  y: -35.146018981933594
  yaw_rad: 3.141592653589793
  yaw_deg: 180.0
- x: 26.0
  y: -35.146018981933594
  yaw_rad: -0.7853981633974483
  yaw_deg: -45.0
- x: 30.0
  y: -39.146018981933594
  yaw_rad: 3.141592653589793
  yaw_deg: 180.0
- x: 26.0
  y: -39.146018981933594
  yaw_rad: -0.7853981633974483
  yaw_deg: -45.0
- x: 30.0
  y: -43.146018981933594
  yaw_rad: 3.141592653589793
  yaw_deg: 180.0
- x: 26.0
  y: -43.146018981933594
  yaw_rad: -0.6713387743353485
  yaw_deg: -38.464878392901056
- x: 30.233135223388672
  y: -46.50897216796875
  yaw_rad: -2.8276302372744118
  yaw_deg: -162.01127861939935
- x: 22.8196434020996
  y: -48.91614837646483
  yaw_rad: 0.0076581009087023436
  yaw_deg: 0.4387768611539449
- x: 31.80655288696289
  y: -48.84732437133789
  yaw_rad: -2.772412336307027
  yaw_deg: -158.84752594039688
- x: 23.851796722412097
  y: -51.9251823425293
  yaw_rad: 0.18398934459939678
  yaw_deg: 10.541812920923562
- x: 34.39466094970703
  y: -49.963218688964844
  yaw_rad: -2.6120796737990952
  yaw_deg: -149.66114106059695
- x: 27.610392761230475
  y: -53.933799743652344
  yaw_rad: 0.3572126915021499
  yaw_deg: 20.466779611581877
- x: 38.15044403076172
  y: -50.0
  yaw_rad: -2.412473026481105
  yaw_deg: -138.22452260651983
- x: 30.650444030761772
  y: -56.699999999999974
  yaw_rad: 0.527533615596452
  yaw_deg: 30.22544972495344
- x: 42.15044403076172
  y: -50.0
  yaw_rad: -2.326791201988337
  yaw_deg: -133.31531568210357
- x: 35.55044403076178
  y: -56.99999999999997
  yaw_rad: 0.5836358190771621
  yaw_deg: 33.43986920578229
- x: 46.15044403076172
  y: -50.0
  yaw_rad: -2.195819046216392
  yaw_deg: -125.8111639226411
- x: 40.45044403076177
  y: -57.89999999999996
  yaw_rad: 0.7047982662375442
  yaw_deg: 40.38196606354902
- x: 49.818050384521484
  y: -49.93262481689453
  yaw_rad: -1.4064469300141227
  yaw_deg: -80.5834731989407
- x: 50.47249221801758
  y: -53.87872314453125
  yaw_rad: 1.2215923250712317
  yaw_deg: 69.99208451215488
- x: 52.35581970214844
  y: -48.706546783447266
  yaw_rad: -0.835018323014114
  yaw_deg: -47.843025724800434
- x: 55.04047393798828
  y: -51.67177963256836
  yaw_rad: 1.7930216949562445
  yaw_deg: 102.73257569638614
- x: 53.827301025390625
  y: -46.30274200439453
  yaw_rad: -0.2635891966539833
  yaw_deg: -15.102548493517123
- x: 57.68914794921875
  y: -47.344932556152344
  yaw_rad: 2.257522361805535
  yaw_deg: 129.3465034878628
- x: 54.0
  y: -42.845130920410156
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: 58.0
  y: -42.845130920410156
  yaw_rad: 2.4858900885464483
  yaw_deg: 142.431010407114
- x: 10.08068561553955
  y: -5.983590602874756
  yaw_rad: -3.0134077494736657
  yaw_deg: -172.65554599685677
- x: 6.882225036621094
  y: -6.395845413208008
  yaw_rad: -2.4481139643428738
  yaw_deg: -140.26659792388716
- x: 4.282883644104004
  y: -8.556421279907227
  yaw_rad: -2.188745226431752
  yaw_deg: -125.4058639039451
- x: 2.4218249320983887
  y: -11.174617767333984
  yaw_rad: -1.7026445651169049
  yaw_deg: -97.55434759208612
- x: 1.9848729372024536
  y: -14.469444274902344
  yaw_rad: -2.2922003328596476
  yaw_deg: -131.3334048713403
- x: -2.0
  y: -19.0
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: 2.0
  y: -19.0
  yaw_rad: 1.0152174676038739
  yaw_deg: 58.167676181661356
- x: 10.08068561553955
  y: -5.983590602874756
  yaw_rad: -3.0134077494736657
  yaw_deg: -172.65554599685677
- x: 6.882225036621094
  y: -6.395845413208008
  yaw_rad: -2.447987599177846
  yaw_deg: -140.2593577332536
- x: 4.221144199371338
  y: -8.608307838439941
  yaw_rad: -2.8467207078830383
  yaw_deg: -163.1050820141922
- x: -0.9213457107543945
  y: -10.170217514038086
  yaw_rad: 2.60394472794582
  yaw_deg: 149.19504299663683
- x: -5.218805313110352
  y: -7.607913494110107
  yaw_rad: 2.6102714450576965
  yaw_deg: 149.55753718532057
- x: -7.780153274536133
  y: -6.102626800537109
  yaw_rad: 0.8287996863767401
  yaw_deg: 47.48672409115348
- x: -0.11479467153549194
  y: 2.258753776550293
  yaw_rad: -1.5508278513678337
  yaw_deg: -88.8558906347186
- x: -0.044909656047821045
  y: -1.2405482530593872
  yaw_rad: -0.3372033074275869
  yaw_deg: -19.320326353453133
- x: 54.0
  y: -20.18833351135254
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: 58.0
  y: -20.18833351135254
  yaw_rad: 2.356194490192345
  yaw_deg: 135.0
- x: 54.0
  y: -16.18833351135254
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: 58.0
  y: -16.18833351135254
  yaw_rad: 2.3829074010091076
  yaw_deg: 136.53053704830987
- x: 53.971553802490234
  y: -12.369559288024902
  yaw_rad: 0.09018535336918582
  yaw_deg: 5.167240121950287
- x: 57.95529556274414
  y: -12.009306907653809
  yaw_rad: 2.660275760014947
  yaw_deg: 152.4225733898139
- x: 53.02320098876953
  y: -9.433343887329102
  yaw_rad: 0.5346290753056695
  yaw_deg: 30.631989619996723
- x: 56.46503448486328
  y: -7.395254611968994
  yaw_rad: 3.104720030801424
  yaw_deg: 177.88735433464853
- x: 50.904541015625
  y: -7.190131664276123
  yaw_rad: 0.9790745558864354
  yaw_deg: 56.0968398809382
- x: 53.135704040527344
  y: -3.870206356048584
  yaw_rad: -2.734020718996115
  yaw_deg: -156.64784829980022
- x: 48.0272216796875
  y: -6.075780868530273
  yaw_rad: 1.4235188051162944
  yaw_deg: 81.56161959066961
- x: 48.61420440673828
  y: -2.1190836429595947
  yaw_rad: -2.406068215396679
  yaw_deg: -137.8575539628036
- x: 44.32550048828125
  y: -6.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: 44.32550048828125
  y: -2.0
  yaw_rad: -2.356194490192345
  yaw_deg: -135.0
- x: 40.32550048828125
  y: -6.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: 40.32550048828125
  y: -2.0
  yaw_rad: -2.356194490192345
  yaw_deg: -135.0
- x: 36.32550048828125
  y: -6.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: 36.32550048828125
  y: -2.0
  yaw_rad: -2.356194490192345
  yaw_deg: -135.0
- x: 32.32550048828125
  y: -6.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: 32.32550048828125
  y: -2.0
  yaw_rad: -2.356194490192345
  yaw_deg: -135.0
- x: 28.32550048828125
  y: -6.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: 28.32550048828125
  y: -2.0
  yaw_rad: -2.356194490192345
  yaw_deg: -135.0
- x: 24.32550048828125
  y: -6.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: 24.32550048828125
  y: -2.0
  yaw_rad: -2.356194490192345
  yaw_deg: -135.0
- x: 20.32550048828125
  y: -6.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: 20.32550048828125
  y: -2.0
  yaw_rad: -0.847429962900525
  yaw_deg: -48.55416030712801
- x: 54.0
  y: -40.134674072265625
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: 58.0
  y: -40.134674072265625
  yaw_rad: 2.356194490192345
  yaw_deg: 135.0
- x: 54.0
  y: -36.134674072265625
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: 58.0
  y: -36.134674072265625
  yaw_rad: 2.356194490192345
  yaw_deg: 135.0
- x: 54.0
  y: -32.134674072265625
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: 58.0
  y: -32.134674072265625
  yaw_rad: 2.356194490192345
  yaw_deg: 135.0
- x: 54.0
  y: -28.134674072265625
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: 58.0
  y: -28.134674072265625
  yaw_rad: 2.356194490192345
  yaw_deg: 135.0
- x: 54.0
  y: -24.134674072265625
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: 58.0
  y: -24.134674072265625
  yaw_rad: 2.8909996517526353
  yaw_deg: 165.64207861921676
- x: -12.911069869995117
  y: -5.983306407928467
  yaw_rad: 0.15241816249215653
  yaw_deg: 8.732917431939756
- x: 12.912003517150879
  y: -2.016636371612549
  yaw_rad: -1.5709646228664915
  yaw_deg: -90.00964265461103
- x: 12.911335945129395
  y: -5.983288764953613
  yaw_rad: -2.459225559338643
  yaw_deg: -140.90324542080344
- x: -2.0
  y: -18.100000381469727
  yaw_rad: 1.567308211858488
  yaw_deg: 89.80014573568725
- x: -1.9863508939743042
  y: -14.186984062194824
  yaw_rad: 1.8394227473201967
  yaw_deg: 105.39116016180613
- x: -2.8284497261047363
  y: -11.127920150756836
  yaw_rad: 2.02005911240151
  yaw_deg: 115.74086150754971
- x: -4.5656561851501465
  y: -7.524849891662598
  yaw_rad: 1.8285012838049106
  yaw_deg: 104.76540639627413
- x: -5.830415725708008
  y: -2.7261977195739746
  yaw_rad: 1.0360417049711892
  yaw_deg: 59.360817094387144
- x: -2.905019998550415
  y: 2.2126646041870117
  yaw_rad: -0.1335627211779089
  yaw_deg: -7.652580223776759
- x: 2.8301100730895996
  y: 1.442077398300171
  yaw_rad: -1.022752623482074
  yaw_deg: -58.59940881145542
- x: 5.719414234161377
  y: -3.291252851486206
  yaw_rad: -1.8998248206064645
  yaw_deg: -108.85194403494917
- x: 4.046995162963867
  y: -8.189391136169434
  yaw_rad: -2.033585490075107
  yaw_deg: -116.51586586034679
- x: 2.504232168197632
  y: -11.281550407409668
  yaw_rad: -1.7243340386976993
  yaw_deg: -98.79706288812613
- x: 1.986552357673645
  y: -14.626692771911621
  yaw_rad: -2.309924398158701
  yaw_deg: -132.34891900879032
- x: -2.0
  y: -19.000349044799805
  yaw_rad: -1.7774820918758076
  yaw_deg: -101.84222202456861
- x: -2.636910915374756
  y: -22.037885665893555
  yaw_rad: -2.231683957455058
  yaw_deg: -127.866071969228
- x: -4.520893096923828
  y: -24.460926055908203
  yaw_rad: -2.683886079318959
  yaw_deg: -153.77534503889007
- x: -7.274252414703369
  y: -25.817218780517578
  yaw_rad: 1.6874922133275378
  yaw_deg: 96.6861817848579
- x: -10.064788818359375
  y: -2.0129711627960205
  yaw_rad: 0.1844495591377791
  yaw_deg: 10.56818127164343
- x: -6.670571327209473
  y: -1.379711389541626
  yaw_rad: 0.4655066591864041
  yaw_deg: 26.671566906615766
- x: -3.5865776538848877
  y: 0.16945874691009521
  yaw_rad: 0.4011189100364636
  yaw_deg: 22.982420627977124
- x: 1.2346892356872559
  y: 2.214219570159912
  yaw_rad: -0.9006502362978325
  yaw_deg: -51.6034573573261
- x: 5.099324703216553
  y: -2.662341356277466
  yaw_rad: -1.6586911915716276
  yaw_deg: -95.03600479257976
- x: 4.71155309677124
  y: -7.062740802764893
  yaw_rad: -2.0468644732415764
  yaw_deg: -117.27669555201075
- x: 2.678722381591797
  y: -11.00521183013916
  yaw_rad: -1.7945325487389145
  yaw_deg: -102.81914124159451
- x: 1.991551399230957
  y: -14.02513599395752
  yaw_rad: -1.5686258849592398
  yaw_deg: -89.87564284313824
- x: 2.0
  y: -17.917701721191406
  yaw_rad: 0.9755996988066683
  yaw_deg: 55.89774523585639
- x: 10.08068561553955
  y: -5.983590602874756
  yaw_rad: -3.0134077494736657
  yaw_deg: -172.65554599685677
- x: 6.882225036621094
  y: -6.395845413208008
  yaw_rad: -2.447987599177846
  yaw_deg: -140.2593577332536
- x: 4.221144199371338
  y: -8.608307838439941
  yaw_rad: -2.8467207078830383
  yaw_deg: -163.1050820141922
- x: -0.9213457107543945
  y: -10.170217514038086
  yaw_rad: 2.547556109186898
  yaw_deg: 145.96421312917838
- x: -5.375276565551758
  y: -7.161953926086426
  yaw_rad: 1.6586476298704147
  yaw_deg: 95.03350889095186
- x: -5.846822261810303
  y: -1.8082261085510254
  yaw_rad: 0.22597449141276638
  yaw_deg: 12.947384635566777
- x: 4.3259596824646
  y: 0.5305080413818359
  yaw_rad: -1.2067083926596596
  yaw_deg: -69.13929800241382
- x: 6.2398271560668945
  y: -4.491748809814453
  yaw_rad: -2.058786594947967
  yaw_deg: -117.95978280862823
- x: 3.8477096557617188
  y: -8.998295783996582
  yaw_rad: -2.1053744634959366
  yaw_deg: -120.62907105293718
- x: 2.3000192642211914
  y: -11.612269401550293
  yaw_rad: -1.6597336131360436
  yaw_deg: -95.09573114869423
- x: 1.9933891296386719
  y: -15.050886154174805
  yaw_rad: -2.2048795546740636
  yaw_deg: -126.3302928175083
- x: -9.0
  y: -30.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: -9.0
  y: -26.0
  yaw_rad: -2.356194490192345
  yaw_deg: -135.0
- x: -13.0
  y: -30.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: -13.0
  y: -26.0
  yaw_rad: -2.356194490192345
  yaw_deg: -135.0
- x: -17.0
  y: -30.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: -17.0
  y: -26.0
  yaw_rad: -2.356194490192345
  yaw_deg: -135.0
- x: -21.0
  y: -30.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: -21.0
  y: -26.0
  yaw_rad: -2.2128824294683573
  yaw_deg: -126.78882376719295
- x: -24.168254852294922
  y: -30.23681640625
  yaw_rad: 1.9707967341548929
  yaw_deg: 112.91833514524146
- x: -25.725929260253906
  y: -26.552574157714844
  yaw_rad: -1.5838892375045874
  yaw_deg: -90.75016852520692
- x: -25.79611587524414
  y: -31.912925720214844
  yaw_rad: 2.7707969632710645
  yaw_deg: 158.75497188309697
- x: -29.524272918701172
  y: -30.463497161865234
  yaw_rad: -0.9256050727518746
  yaw_deg: -53.03326416458193
- x: -26.0
  y: -35.146018981933594
  yaw_rad: 3.141592653589793
  yaw_deg: 180.0
- x: -30.0
  y: -35.146018981933594
  yaw_rad: -0.7853981633974483
  yaw_deg: -45.0
- x: -26.0
  y: -39.146018981933594
  yaw_rad: 3.141592653589793
  yaw_deg: 180.0
- x: -30.0
  y: -39.146018981933594
  yaw_rad: -0.7853981633974483
  yaw_deg: -45.0
- x: -26.0
  y: -43.146018981933594
  yaw_rad: 3.141592653589793
  yaw_deg: 180.0
- x: -30.0
  y: -43.146018981933594
  yaw_rad: -0.9062437181715188
  yaw_deg: -51.92394026147126
- x: -26.41964340209961
  y: -47.716148376464844
  yaw_rad: 2.8350187076488664
  yaw_deg: 162.43460678891304
- x: -30.233135223388672
  y: -46.50897216796875
  yaw_rad: -1.391555375392107
  yaw_deg: -79.73024996871067
- x: -29.25179672241211
  y: -51.9251823425293
  yaw_rad: 2.2635899491450213
  yaw_deg: 129.69415063424236
- x: -31.80655288696289
  y: -48.84732437133789
  yaw_rad: -1.9629842176620151
  yaw_deg: -112.47071092282322
- x: -33.91039276123047
  y: -53.93379592895508
  yaw_rad: 1.6921612112018065
  yaw_deg: 96.95369565760903
- x: -34.39466094970703
  y: -49.96322250366211
  yaw_rad: -2.320150789604314
  yaw_deg: -132.9348480782726
- x: -38.15044403076172
  y: -54.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: -38.15044403076172
  y: -50.0
  yaw_rad: -2.356194490192345
  yaw_deg: -135.0
- x: -42.15044403076172
  y: -54.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: -42.15044403076172
  y: -50.0
  yaw_rad: -2.356194490192345
  yaw_deg: -135.0
- x: -46.15044403076172
  y: -54.0
  yaw_rad: 1.5707963267948966
  yaw_deg: 90.0
- x: -46.15044403076172
  y: -50.0
  yaw_rad: -2.41020086681928
  yaw_deg: -138.09433744751735
- x: -50.47249221801758
  y: -53.87872314453125
  yaw_rad: 1.4064469300141227
  yaw_deg: 80.5834731989407
- x: -49.818050384521484
  y: -49.93262481689453
  yaw_rad: -2.8201270000947147
  yaw_deg: -161.58137479631708
- x: -55.04047393798828
  y: -51.67177963256836
  yaw_rad: 0.835018323014114
  yaw_deg: 47.843025724800434
- x: -52.35581970214844
  y: -48.706546783447266
  yaw_rad: 2.8916292762997244
  yaw_deg: 165.67815344844283
- x: -57.68914794921875
  y: -47.344932556152344
  yaw_rad: 0.2635891966539833
  yaw_deg: 15.102548493517123
- x: -53.827301025390625
  y: -46.30274200439453
  yaw_rad: 2.4496383148941074
  yaw_deg: 140.3539367769713
- x: -58.0
  y: -42.845130920410156
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: -54.0
  y: -42.845130920410156
  yaw_rad: 2.356194490192345
  yaw_deg: 135.0
- x: -58.0
  y: -38.845130920410156
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: -54.0
  y: -38.845130920410156
  yaw_rad: 2.356194490192345
  yaw_deg: 135.0
- x: -58.0
  y: -34.845130920410156
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: -54.0
  y: -34.845130920410156
  yaw_rad: 2.356194490192345
  yaw_deg: 135.0
- x: -58.0
  y: -30.845130920410156
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: -54.0
  y: -30.845130920410156
  yaw_rad: 2.356194490192345
  yaw_deg: 135.0
- x: -58.0
  y: -26.845130920410156
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: -54.0
  y: -26.845130920410156
  yaw_rad: 2.356194490192345
  yaw_deg: 135.0
- x: -58.0
  y: -22.845130920410156
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: -54.0
  y: -22.845130920410156
  yaw_rad: 2.356194490192345
  yaw_deg: 135.0
- x: -58.0
  y: -18.845130920410156
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: -54.0
  y: -18.845130920410156
  yaw_rad: 2.356194490192345
  yaw_deg: 135.0
- x: -58.0
  y: -14.845130920410156
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: -54.0
  y: -14.845130920410156
  yaw_rad: 2.334336884037936
  yaw_deg: 133.7476514170932
- x: -57.950103759765625
  y: -10.718460083007812
  yaw_rad: -0.06381691015536615
  yaw_deg: -3.6564396134680432
- x: -53.95824432373047
  y: -10.973554611206055
  yaw_rad: 2.073939689212353
  yaw_deg: 118.82799115654146
- x: -56.687374114990234
  y: -6.015025615692139
  yaw_rad: -0.5206310958312873
  yaw_deg: -29.82996447440387
- x: -53.217350006103516
  y: -8.004737854003906
  yaw_rad: 1.4504558302711745
  yaw_deg: 83.10499744468197
- x: -52.57218933105469
  y: -2.669515609741211
  yaw_rad: -1.2320994231782982
  yaw_deg: -70.59409688861969
- x: -51.24315643310547
  y: -6.442268371582031
  yaw_rad: 0.9382290819587146
  yaw_deg: 53.75656661266815
- x: -47.989864349365234
  y: -2.0042712688446045
  yaw_rad: -1.558016742333092
  yaw_deg: -89.26778374640763
- x: -47.93874740600586
  y: -6.003944396972656
  yaw_rad: 0.7890979153884423
  yaw_deg: 45.211980174329085
- x: -43.96432113647461
  y: -2.0
  yaw_rad: -1.5707963267948966
  yaw_deg: -90.0
- x: -43.96432113647461
  y: -6.0
  yaw_rad: 0.7853981633974483
  yaw_deg: 45.0
- x: -39.96432113647461
  y: -2.0
  yaw_rad: -1.5707963267948966
  yaw_deg: -90.0
- x: -39.96432113647461
  y: -6.0
  yaw_rad: 0.7853981633974483
  yaw_deg: 45.0
- x: -35.96432113647461
  y: -2.0
  yaw_rad: -1.5707963267948966
  yaw_deg: -90.0
- x: -35.96432113647461
  y: -6.0
  yaw_rad: 0.7853981633974483
  yaw_deg: 45.0
- x: -31.96432113647461
  y: -2.0
  yaw_rad: -1.5707963267948966
  yaw_deg: -90.0
- x: -31.96432113647461
  y: -6.0
  yaw_rad: 0.785397924978926
  yaw_deg: 44.99998633962492
- x: -27.964319229125977
  y: -2.0
  yaw_rad: -1.5707963267948966
  yaw_deg: -90.0
- x: -27.964319229125977
  y: -6.0
  yaw_rad: 0.7853981633974483
  yaw_deg: 45.0
- x: -23.964319229125977
  y: -2.0
  yaw_rad: -1.5707963267948966
  yaw_deg: -90.0
- x: -23.964319229125977
  y: -6.0
  yaw_rad: 0.7853981633974483
  yaw_deg: 45.0
- x: -19.964319229125977
  y: -2.0
  yaw_rad: -1.5707963267948966
  yaw_deg: -90.0
- x: -19.964319229125977
  y: -6.0
  yaw_rad: 0.39500591845363914
  yaw_deg: 22.632172010082282
- x: -4.310125350952148
  y: 0.5265274047851562
  yaw_rad: -0.8076203393491628
  yaw_deg: -46.273236893630354
- x: -1.890855073928833
  y: -2.00272798538208
  yaw_rad: -3.1403322517232226
  yaw_deg: -179.9277842925551
- x: -12.911733627319336
  y: -2.0166187286376953
  yaw_rad: -0.13195465444901724
  yaw_deg: -7.560444787035859
- x: 17.10035514831543
  y: -6.0
  yaw_rad: 3.137964862061116
  yaw_deg: 179.7921428564535
- x: 13.10643482208252
  y: -5.98551082611084
  yaw_rad: -1.7731610384738898
  yaw_deg: -101.59464390158807
- x: 9.0
  y: -26.0
  yaw_rad: 3.141592653589793
  yaw_deg: 180.0
- x: 5.0
  y: -26.0
  yaw_rad: 3.141592653589793
  yaw_deg: 180.0
- x: 1.0000001192092896
  y: -26.0
  yaw_rad: 3.141592653589793
  yaw_deg: 180.0
- x: -2.999999761581421
  y: -26.0
  yaw_rad: 3.141592653589793
  yaw_deg: 180.0
- x: -7.0
  y: -26.0
  yaw_rad: -2.0344439357957027
  yaw_deg: -116.56505117707799
- x: -9.0
  y: -30.0
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: -5.0
  y: -30.0
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: -1.0
  y: -30.0
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: 3.0
  y: -30.0
  yaw_rad: 0.0
  yaw_deg: 0.0
- x: 7.0
  y: -30.0
  yaw_rad: 1.1072185291906684
  yaw_deg: 63.438948721307845
- x: 8.999650955200195
  y: -26.0
  yaw_rad: 2.9349066750790858
  yaw_deg: 168.15776574680484
- x: 5.962115287780762
  y: -25.363088607788086
  yaw_rad: 2.48070592224764
  yaw_deg: 142.13397955789833
- x: 3.5390734672546387
  y: -23.479108810424805
  yaw_rad: 2.0285026909840824
  yaw_deg: 116.22464292431815
- x: 2.18278169631958
  y: -20.725749969482422
  yaw_rad: 2.750285053073208
  yaw_deg: 157.57972599900845
- x: -2.0
  y: -19.0
  yaw_rad: -1.5495802738599251
  yaw_deg: -88.7844097089
- x: -1.9087589979171753
  y: -23.29991912841797
  yaw_rad: -1.0794852369922543
  yaw_deg: -61.84994812633562
- x: 0.5702246427536011
  y: -27.932897567749023
  yaw_rad: -0.4011806451274676
  yaw_deg: -22.98595778813951
- x: 5.407527446746826
  y: -29.984811782836914
  yaw_rad: 2.1281555410939896
  yaw_deg: 121.93433065206564
- x: -2.0
  y: -18.100000381469727
  yaw_rad: 1.567308211858488
  yaw_deg: 89.80014573568725
- x: -1.9863508939743042
  y: -14.186984062194824
  yaw_rad: 1.7151136474732602
  yaw_deg: 98.26877338550632
- x: -2.503873348236084
  y: -10.625910758972168
  yaw_rad: 1.9118881043306712
  yaw_deg: 109.54311927941507
- x: -3.7102513313293457
  y: -7.227334976196289
  yaw_rad: 1.9591359744241374
  yaw_deg: 112.25022282675307
- x: -5.488449573516846
  y: -2.880887746810913
  yaw_rad: 0.8547818332818478
  yaw_deg: 48.97539145150504
- x: -1.3227083683013916
  y: 1.9070944786071777
  yaw_rad: -0.12769487594592038
  yaw_deg: -7.316377457147854
- x: 3.6059958934783936
  y: 1.2742809057235718
  yaw_rad: -0.7196271758342199
  yaw_deg: -41.231599998219586
- x: 6.7365264892578125
  y: -1.4693427085876465
  yaw_rad: -0.17789415189946883
  yaw_deg: -10.19258410389874
- x: 9.646952629089355
  y: -1.9926221370697021
  yaw_rad: -0.0009898647633181136
  yaw_deg: -0.056715073226844055
- x: 17.10035514831543
  y: -2.0
  yaw_rad: -3.1379701567709737
  yaw_deg: -179.79244622098207
- x: 13.106783866882324
  y: -2.0144667625427246
  yaw_rad: -2.324826871381636
  yaw_deg: -133.20276782877121
- x: -2.0
  y: -18.100000381469727
  yaw_rad: 1.567308211858488
  yaw_deg: 89.80014573568725
- x: -1.9863508939743042
  y: -14.186984062194824
  yaw_rad: 1.7254632477591894
  yaw_deg: 98.86176180153745
- x: -2.493081569671631
  y: -10.936880111694336
  yaw_rad: 2.224914792348173
  yaw_deg: 127.47822737777625
- x: -4.501351356506348
  y: -8.317591667175293
  yaw_rad: 2.479600015166888
  yaw_deg: 142.07061574963762
- x: -7.086612224578857
  y: -6.302890777587891
  yaw_rad: -1.1637075118427376
  yaw_deg: -66.67552901625913
- x: -2.0
  y: -18.100000381469727
  yaw_rad: 1.567308211858488
  yaw_deg: 89.80014573568725
- x: -1.9863508939743042
  y: -14.186984062194824
  yaw_rad: 1.7254632477591894
  yaw_deg: 98.86176180153745
- x: -2.493081569671631
  y: -10.936880111694336
  yaw_rad: 2.226089533463163
  yaw_deg: 127.54553508568569
- x: -4.535061836242676
  y: -8.280094146728516
  yaw_rad: 1.882899707016441
  yaw_deg: 107.88220645846131
- x: -6.185347557067871
  y: -3.165280818939209
  yaw_rad: 0.0
  yaw_deg: 0.0
